core/chains/evm/client/node_lifecycle.go

```diff
@@ -151,13 +156,38 @@ func (n *node) aliveLoop() {
 			}
 			if pollFailureThreshold > 0 && pollFailures >= pollFailureThreshold {
 				lggr.Errorw(fmt.Sprintf("RPC endpoint failed to respond to %d consecutive polls", pollFailures), "pollFailures", pollFailures, "nodeState", n.State())
-				if n.nLiveNodes != nil && n.nLiveNodes() < 2 {
-					lggr.Critical("RPC endpoint failed to respond to polls; but cannot disable this connection because there are no other RPC endpoints, or all other RPC endpoints are dead. Chainlink is now operating in a degraded state and urgent action is required to resolve the issue")
-					continue
+				if n.nLiveNodes != nil {
+					if l, _, _ := n.nLiveNodes(); l < 2 {
+						lggr.Criticalf("RPC endpoint failed to respond to polls; %s %s", msgCannotDisable, msgDegradedState)
+						continue
+					}
 				}
 				n.declareUnreachable()
 				return
 			}
+			//TODO add tests
+			if threshold := n.cfg.NodeSyncThreshold(); threshold > 0 && n.nLiveNodes != nil {
```

Comment:
This appears to rely on the poll ticker... would it be preferable to have a channel that fires when the latest head in the pool is updated? Since polling can theoretically be disabled.